app/homepage/page.tsx

```diff
@@ -1,116 +1,117 @@
-'use client'
-import React, { useState } from 'react';
-import Image from 'next/image';
-import logo from '../../public/images/logo.png';
-import { ChevronLeft, Trash2 } from 'lucide-react';
-import Link from 'next/link';
-import { Dialog, DialogTrigger, DialogContent, DialogTitle } from "@/components/ui/dialog";
+  'use client'
+  import React, { useState } from 'react';
+  import Image from 'next/image';
+  import logo from '../../public/images/logo.png';
+  import { ChevronLeft, Trash2 } from 'lucide-react';
+  import Link from 'next/link';
+  import { Dialog, DialogTrigger, DialogContent, DialogTitle } from "@/components/ui/dialog";
 
-import { Input } from '@/components/ui/input';
-import { Button } from '@/components/ui/button';
-import paper from '../../public/images/paper.png';
+  import { Input } from '@/components/ui/input';
+  import { Button } from '@/components/ui/button';
+  import paper from '../../public/images/paper.png';
 
-const LandingPage = () => {
-  const [projectName, setProjectName] = useState('');
-  const [projects, setProjects] = useState([]);
-  
-  const [isDialogOpen, setIsDialogOpen] = useState(false);
-
-  const handleSubmit = (event: { preventDefault: () => void; }) => {
-    event.preventDefault();
-
-    const newProject = {
-        id: projectName,
+  const LandingPage = () => {
+    const [projectName, setProjectName] = useState('');
+    const [projects, setProjects] = useState([]);
+    
+    const [isDialogOpen, setIsDialogOpen] = useState(false);
+    const handleSubmit = (event: { preventDefault: () => void; }) => {
+      event.preventDefault();
+    
+      const newProject = {
+        id: new Date().getTime(),
         name: projectName,
       };
+    
+      setProjects([...projects, newProject]);
+    
+      setProjectName('');
+    
+      setIsDialogOpen(false);
+    };
+    
 
-    setProjects([...projects, projectName]);
-  
-    setProjectName('');
+    const handleRemoveProject = (projectId) => {
+      setProjects((prevProjects) => prevProjects.filter((project) => project.id !== projectId));
+    };
 
-    setIsDialogOpen(false);
-  };
 
-  const handleRemoveProject = (projectId) => {
-    setProjects((prevProjects) => prevProjects.filter((project) => project.id !== projectId));
-  };
+    return (
+      <div className='bg-[#00040D] w-screen h-screen relative flex justify-center items-start pt-48 text-white'>
+        <Link href='/login' className='absolute top-0 left-0 mt-12 ml-16 flex items-center'>
+          <ChevronLeft className='size-8' /> <Image src={logo} alt='' className='size-16' />
+        </Link>
+        <div className='absolute top-0 left-0 w-full flex justify-center pt-14'>
```

Comment:
try to not use absolute.

use flexbox instead

Reply:
https://github.com/21ftt1750/tododo/pull/2/commits/9fdd58390e30c01905e008d61ccde062f78f372b

fixed